prometheus-net/Histogram.cs

```diff
@@ -48,55 +47,35 @@ internal Histogram(string name, string help, string[] labelNames, double[] bucke
 
         public class Child : Advanced.Child, IHistogram
         {
-            private double _sum = 0;
-            private ulong _count = 0;
```

Comment:
This can and should be computed during collection as you ideally want the total could on the result of the collection to match the count that makes up the buckets. If you have the count computed here like this then it can continue to update while other metrics are being collected if we remove locks (which we should as recommended by prometheus client library writing documentation)